### PR TITLE
Add WinDiff

### PR DESCRIPTION
**Pull request by parthopdas**

### CODE REVIEW

File: bucket/windiff.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+    "version": "2.1.0",
+    "description": "The omnipotent, omniscient, omnipresent diffing program",
+    "homepage": "https://en.wikipedia.org/wiki/WinDiff",
+    "license": "UNLICENSED",
+    "architecture": {
```
Comment (rasa) [inline bucket/windiff.json:6]: 32bit programs can run on 64bit, so the architecture tag can be shortened to:
```suggestion
        "url": "https://github.com/kitamstudios/WinDiff/releases/download/refs%2Fheads%2Fmaster/WinDiff.zip",
        "hash": "e25ccdc2fef77eb001fc3a5d755d448a301338d7e4feb2cf365bcb3d34200931",
```
Also can you add an `autoupdate`?

Comment (parthopdas) [reply bucket/windiff.json]: thanks for you review :)

i dont have the code building yet. so it is just the checked in version. i was planning to make the auto update once that work is done.

let me know if that is not ok with you.

**Comment by parthopdas** [comment]
@rasa thank you for fixing windiff after my change. please let me know if i need to do anything else before merging this.

File: bucket/windiff.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    "description": "The omnipotent, omniscient, omnipresent diffing program",
+    "homepage": "https://en.wikipedia.org/wiki/WinDiff",
+    "license": "UNLICENSED",
+    "url": "https://github.com/kitamstudios/WinDiff/releases/download/v2.1.0/WinDiff.zip",
+    "hash": "e25ccdc2fef77eb001fc3a5d755d448a301338d7e4feb2cf365bcb3d34200931",
+    "bin": "bin\\WinDiff.exe",
+    "checkver": {
+        "github": "https://github.com/kitamstudios/WinDiff",
+        "regex": "/releases/tag/v([\\d.]+)"
+    }
```
Comment (rasa) [inline bucket/windiff.json:12]:
```suggestion
    },
    "autoupdate": {
        "url": "https://github.com/kitamstudios/WinDiff/releases/download/v$version/WinDiff.zip"
    }
```

**Review by rasa — CHANGES_REQUESTED** [review]

File: bucket/windiff.json (hunk 2 above)
Comment (parthopdas) [reply bucket/windiff.json:15]: verified that both checkver and autoupdate urls resolve correctly with the change.

**Comment by parthopdas** [comment]
@rasa not sure of the process here as i have taken in the changes + verified but the issues is still marked "waiting feedback". i cannot unmark it. so please let me know if there is anything else that needs to be done.

**Review by rasa — APPROVED** [review]

**Comment by rasa** [comment]
I am going to commit this to the extras bucket, as it's a GUI app, per our bucket [criteria](https://github.com/lukesampson/scoop/wiki/Criteria-for-including-apps-in-the-main-bucket)

**Comment by parthopdas** [comment]
> I am going to commit this to the extras bucket, as it's a GUI app, per our bucket [criteria](https://github.com/lukesampson/scoop/wiki/Criteria-for-including-apps-in-the-main-bucket)

Thanks for doing so. I'd have done it myself if I had known. I wonder if using the github templates it is possible to automatically make contributors aware of these things.

**Pull request closed without merging**